main_test.go

```diff
@@ -116,20 +116,8 @@ func generatedFiles(t *testing.T, dir string) (map[string]string, error) {
 	return files, err
 }
 
-func TestGenerateError(t *testing.T) {
+func TestGenerateScripts(t *testing.T) {
 	testscript.Run(t, testscript.Params{
 		Dir: filepath.Join("testdata", "scripts"),
-		Setup: func(env *testscript.Env) error {
```

Comment:
This is no longer required because the .gunkconfig is being written as a file in the testscript, or it is using the default config.